frontend/src/lib/components/LemonSnack/LemonSnack.tsx

```diff
@@ -0,0 +1,32 @@
+import { LemonButton } from '@posthog/lemon-ui'
+import clsx from 'clsx'
+import React from 'react'
+import { IconClose } from '../icons'
+import './LemonSnack.scss'
+
+export interface LemonSnackProps {
+    children?: React.ReactNode
+    disabled?: boolean
+    onClose?: () => void
+    'data-attr'?: string
+}
+
+export function LemonSnack({ children, disabled, onClose }: LemonSnackProps): JSX.Element {
```

Comment:
It'd be amazing if `LemonTag` had these options (for instance, we could finally move insights/dashboard tags over to it) so I definitely think this would be great unified as `LemonTag` (also, I know that "snack" is a term of art in nutritional science, but not so sure about UI design). Though these tags definitely has larger dimensions – maybe @clarkus might be able to offer guidance on this? (or a counter-argument)

Reply:
I think we can merge the two if we agree on some common stylings.
I'd also be keen to hear @clarkus thoughts and in particular I wonder if there maybe already exists somewhere in Figma some sort of view of the different types of Snack/Tag 🤞